pages/validator/external-chains/optimism.md

```diff
@@ -0,0 +1,440 @@
+import CodeBlock from '../../../components/code-block'
+import { NextraTab, NextraTabs } from 'nextra-theme-docs'
+
+Instructions to set up your Optimism node.
+
+## Requirements
+
+- [Setup your Axelar validator](/validator/setup)
+- Minimum hardware requirements: 8 AWS vCPU+, 16GB RAM, 80GB+ free storage space.
+- MacOS or Ubuntu 20.04 (tested on 20.04)
+- [Official Documentation](https://community.optimism.io/docs/developers/build/run-a-node/#)
+
+
+## Prerequisites
+```bash
+sudo apt-get install jq -y
+snap install docker 
+```
+
+## Clone Repo
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/smartcontracts/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+git clone https://github.com/ethereum-optimism/simple-optimism-node.git
+cd simple-optimism-node
+cp .env.example .env
+`}
+</CodeBlock>
+}
+]} />
+
+
+## Set variables
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+NODE_TYPE=full
+# can be l1 or l2
+SYNC_SOURCE=
+#  l2 node to check blocks against
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=
+# l1 node RPC to check state roots against
+FAULT_DETECTOR__L1_RPC_PROVIDER=
+# can be l1 node or l2 node, depends on what sync source you have mentioned in the variable SYNC_SOURCE
+DATA_TRANSPORT_LAYER__RPC_ENDPOINT=
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+# only goerli is supported for now
+NETWORK_NAME=goerli
+# can be full or archive
+NODE_TYPE=full
+# only download option is available for now
+BEDROCK_SOURCE=download
+# L1 node that the op-node will get chain data from - if you are using third party - make sure it will not get rate limited
+OP_NODE__RPC_ENDPOINT=
+# Use basic for OP_NODE__RPC_TYPE if you are using your own node in OP_NODE__RPC_ENDPOINT
+# Other options are: alchemy, quicknode, infura, parity, nethermind, debug_geth, erigon
+# Depending upon which provider you are using in OP_NODE__RPC_ENDPOINT, mentioning the right
+# OP_NODE__RPC_TYPE will optimize the process
+OP_NODE__RPC_TYPE=
+`}
+</CodeBlock>
+}
+]} />
+
+#### Example Variables
+These examples include 3rd party RPCs and are just to give you an idea, if you decide to use them, remember to make sure you are not getting rate limited when syncing
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=mainnet
+NODE_TYPE=full
+SYNC_SOURCE=l1
+HEALTHCHECK__REFERENCE_RPC_PROVIDER=https://mainnet.optimism.io
+FAULT_DETECTOR__L1_RPC_PROVIDER=https://mainnet.infura.io/v3/your_infura_key
+DATA_TRANSPORT_LAYER__RPC_ENDPOINT=https://mainnet.infura.io/v3/your_infura_key
+`}
+</CodeBlock>
+},
+{
+title: "Testnet",
+content: <CodeBlock language="bash">
+{`
+NETWORK_NAME=goerli
+NODE_TYPE=full
+BEDROCK_SOURCE=download
+OP_NODE__RPC_ENDPOINT=https://goerli.infura.io/v3/your_infura_key
```

Comment:
Is this the right endpoint for Optimism Goerli? Shouldn't it be something like  those listed here: https://community.optimism.io/docs/useful-tools/networks/#api-options?

Reply:
Yes, it should be a L1 node not optimism node